Лабораторная 3

LeeryLis opened the pull request.

LeeryLis committed:
Лабораторная 3

LeeryLis committed:
Update main.py

Добавлены проверки для атрибутов pages и duration

vladuxa23 reviewed (approved):
Всё отлично, за исключеним отсутствия свойств для атрибутов name и author, конректно `Атрибуты name и author изменяться не могут, поэтому напишите для них свойства, которые не позволят изменять эти атрибуты.` 
Пока 7 из 10

LeeryLis committed:
Update main.py

Добавление свойств для атрибутов name и author (геттеры). 
Использование геттеров и сеттеров для атрибутов pages и duration, что сделало код более логичным. Теперь проверка значений происходит каждый раз при их установке (проверка в сеттере). До этого вызов _init_(название атрибута) для проверки и установки значения происходил только при инициализации нового экземпляра класса.

vladuxa23 reviewed (approved):
Всё хорошо
